core/src/autogluon/core/models/ensemble/fold_fitting_strategy.py

```diff
@@ -408,7 +408,9 @@ def _ray_fit(
     y: Union[str, pd.DataFrame],
     X_pseudo: Union[str, pd.DataFrame],
     y_pseudo: Union[str, pd.DataFrame],
+    task_id: int,
     fold_ctx: Dict[str, Any],
+    assignments: Dict[int, List[int]],
```

Comment:
`assignments` is too generic of a name. Consider something like `task_to_gpu_ids_map`.

Also, why are we sending in a dict when this is a fit call for a specific task_id.
Why not just pass in `gpu_ids` directly?